includes/class-ur-ajax.php

```diff
@@ -976,7 +976,8 @@ public static function sweep_array( &$array ) {
 						$value = wp_kses_post( $value );
 					}
 				} else {
-					$value = sanitize_text_field( $value );
+					$value = str_replace( '"', "'", $value );
+					$value = wp_kses_post( $value );
```

Comment:
Please check if the key is of checkbox field and then handle it like this separately, don't let your code affect fields other than checkbox field.